README.md

```diff
@@ -1,29 +1,125 @@
+
+This is a template app using the GOV.UK Frontend and GOV.UK Design System which is designed to get a new project started quicker.
+
+The app is provided intentionally bare, with just the essential parts that all services need, such as error pages, accessibility statement and privacy notice. It uses a number of other packages to provide the [features](#features) described below with sensible and best-practice defaults. Please read the next steps section for guidance on how to start building out your app on top of this template.
+
+## Prerequisites
+- node stable version [20.15.1]
 
 ## Getting started
 
+### Set local environment variables
+
 Create your local config file `.env` from the template file:
 
 ```shell
 cp .env.example .env
 ```
 
-Install and run application
+### Install and run application
 ```
 npm install
 npm run dev
 ```
-
 Then, load http://localhost:3000/ in your browser to access the app.
 
+#### TO NOTE:
+
+You may have to tell your local machine to use the latest version of node already installed on your device, before installing and running the application. Use the following command.
+
+```
+nvm install node
+```
+
 ### Running the app locally with DEBUG enabled
 
-`DEBUG=moj-express-skeleton:* npm start`
+```
+DEBUG=moj-express-skeleton:* npm start
+```
+This enables DEBUG, and you can type http://localhost:3000/ into your browser to access the app.
+
+## Routing
+TODO
+
+## Testing
+TODO
+
+### Unit testing
+TODO
+
+### e2e testing
+TODO
 
-Then, load http://localhost:3000/ in your browser to access the app.
 
-### Apps Security
+## Features
+- [Asset management](#asset-management)
+- [Cache busting](#cache-busting)
+- [Form validation](#form-validation)
+- [CSRF protection](#csrf-protection)
+- [Content Security Policy (CSP)](#content-security-policy-csp)
+- [Response compression](#response-compression)
+- [Rate limiting](#rate-limiting)
+- [Nunjucks support](#nunjucks-support)
+- [ES6 JSDocs](#es6-jsdocs)
+- [Linter](#linter)
+- [Security](#security)
+- [Axios](#axios)
+- [SQLite database](#sqlite-database)
+
+## Examples of features 
+Please refer to the specific packages documentation for more details.
+
+### Cache busting
+TODO
+
+### Form validation
+TODO
+
+### CSRF protection
+TODO
+
+### Content Security Policy (CSP)
+TODO
+
+### Response compression
+TODO
+
+### Rate limiting
+TODO
+
+### Nunjucks support
+TODO
+
+### ES6 JSDocs
+TODO
+
+### Linter
+TODO
+
+### Security
+TODO
 
 - [helmet.js](https://helmetjs.github.io/)
-- [express-session](https://www.npmjs.com/package/express-session)
\ No newline at end of file
+- [express-session](https://www.npmjs.com/package/express-session)
+
+### Axios
+TODO
+
+### SQLite database
+TODO
+
+
+## Contributers
+- [Patrick Sweeting](https://github.com/psweeting1) (Primary maintatiner)
```

Comment:
Minor spelling mistake - should be "maintainer" for the four contributers.

Reply:
good spot